test/sql/updates/post.continuous_aggs.sql

```diff
@@ -18,11 +18,6 @@ CALL refresh_continuous_aggregate('mat_before',NULL,NULL);
 --the max of the temp for the POR should now be 165
 SELECT * FROM mat_before ORDER BY bucket, location;
 
-SET ROLE cagg_user;
```

Comment:
Looks like all the cagg_user specific tests have been removed. Is the intent to remove these from the update tests? (Not sure why they were here in the first place?)

Reply:
The intention was to remove them, yes, but digging around a little I realized that it makes sense to actually grant privileges that are non-superuser for the update tests so added back the grants under conditions. They were added to check that update tests actually "fix" the grants for continuous aggregates during update, but we have a check of the privileges last so that is covered without the select that is removed.